lib/haml/helpers/safe_erubi_template.rb

```diff
@@ -15,13 +15,5 @@ def prepare
       @options.merge! engine_class: Haml::ErubiTemplateHandler
       super
     end
-
-    def precompiled_preamble(locals)
-      [super, "@output_buffer = ActionView::OutputBuffer.new;"].join("\n")
-    end
-
-    def precompiled_postamble(locals)
-      [super, '@output_buffer.to_s'].join("\n")
```

Comment:
See commit message written in https://github.com/haml/haml/pull/914/commits/596485a7d274ba63ebe23da73058deb76f320d16. These preamble and postamble should be removed for Erubi.